Cookbook/Technical-Documents/ReleaseProcess.md

```diff
@@ -38,7 +38,7 @@ There are usually two release engineers working at any given time. It goes witho
 
 1. Release branch is cut automatically on last Friday of the sprint during nightly builds. It will create and push release branch, create a release Slack channel, submit a new AppCenter build, and run UI tests.
    * if, for any reason, the automatic cut didn't work, it can be triggered manually with `/stevenson release_cutoff target:Babylon version:4.1.0` from `#ios-build` channel
-   * If you're releasing another app (e.g. Telus), since they typically go thru the release process _only_ after the main Babylon app has been signed off by QA, you should create the new branch from the corresponding Babylon release branch that was recently already QA'd and signed off (e.g. `release/babylon/4.1.0`) instead of `develop`, and name your new branch using the same `release/{appname}/{version}` convention (e.g. `release/telus/4.1.0`)
+   * If you're releasing another app (e.g. Telus), since they typically go thru the release process _only_ after the main Babylon app has been signed off by QA, you should create the new branch from the corresponding Babylon release branch that was recently already QA'd and signed off (e.g. `release/babylon/4.1.0`) instead of `develop`, and name your new branch using the same `release/{appname}/{version}` convention (e.g. `release/telus/4.1.0`). To do this, run `/stevenson release_cutoff branch:release/babylon/4.1.0 target:Telus`
```

Comment:
iinm, by default, `release_cutoff` will bump the minor version unless you specify a version explicitly. So I think that this command will bump the version to 4.2.0 once it cut the release/telus branch

```suggestion
   * If you're releasing another app (e.g. Telus), since they typically go thru the release process _only_ after the main Babylon app has been signed off by QA, you should create the new branch from the corresponding Babylon release branch that was recently already QA'd and signed off (e.g. `release/babylon/4.1.0`) instead of `develop`, and name your new branch using the same `release/{appname}/{version}` convention (e.g. `release/telus/4.1.0`). To do this, run `/stevenson release_cutoff branch:release/babylon/4.1.0 version:4.1.0 target:Telus` (you need to specify both the branch and the version so that the script don't bump the version to 4.2.0 by default when cutting the branch)
```